### PR TITLE
Fix dataset notebook link

### PR DESCRIPTION
**Pull request by awf**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ We also provide a lightweight implementation of the SCF algorithm, optimized for
 
 To run the notebook on Graphcore IPU hardware on Paperspace:
 
-[![Run on Gradient](https://assets.paperspace.io/img/gradient-badge.svg)](https://ipu.dev/ipobmC)
+[![Run on Gradient](https://assets.paperspace.io/img/gradient-badge.svg)](https://console.paperspace.com/github/graphcore-research/pyscf-ipu?container=graphcore%2Fpytorch-jupyter%3A3.2.0-ubuntu-20.04&machine=Free-IPU-POD4&file=%2Fnotebooks%2FDFT-dataset-generation.ipynb)
```
Comment (hatemhelal) [inline README.md:43]: presumably this ipu.dev link launches the nanoDFT notebook?

I think we might need a new ipu.dev link to the dataset notebook and link that above on line 34.

Comment (awf) [inline README.md:43]:
```suggestion
[![Run on Gradient](https://assets.paperspace.io/img/gradient-badge.svg)](https://ipu.dev/YX0jlK)

```

File: README.md [hunk 2]
```diff
@@ -40,7 +40,8 @@ We also provide a lightweight implementation of the SCF algorithm, optimized for
 
 To run the notebook on Graphcore IPU hardware on Paperspace:
 
-[![Run on Gradient](https://assets.paperspace.io/img/gradient-badge.svg)](https://console.paperspace.com/github/graphcore-research/pyscf-ipu?container=graphcore%2Fpytorch-jupyter%3A3.2.0-ubuntu-20.04&machine=Free-IPU-POD4&file=%2Fnotebooks%2FDFT-dataset-generation.ipynb)
+[![Run on Gradient](https://assets.paperspace.io/img/gradient-badge.svg)](https://ipu.dev/YX0jlK)
```
Comment (hatemhelal) [inline README.md:43]: I think the URL need swapping to match the section headings (I can't suggest the change for line 34)

```suggestion
[![Run on Gradient](https://assets.paperspace.io/img/gradient-badge.svg)](https://ipu.dev/ipobmC))

```

Comment (awf) [reply README.md:43]: Eek, thanks!

**Review by mihaipgc — APPROVED** [review]
LGTM

**Pull request merged**